pkts-core/src/main/java/io/pkts/framer/IPv4Framer.java

```diff
@@ -47,7 +47,6 @@ public IPPacket frame(final MACPacket parent, final Buffer payload) throws IOExc
         // final int version = ((i >>> 28) & 0x0F);
         // final int length = ((i >>> 24) & 0x0F);
 
-        final int version = b >>> 5 & 0x0F;
```

Comment:
Just comment out in case I will need it back. Just nice to have all the fields defined...